### PR TITLE
Replace tee -a with just tee in Centos/RHEL docs

### PR DESCRIPTION
**Pull request by mbacchi**
## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

### CODE REVIEW

**Comment by mbacchi** [comment]
Should look better now @mihaiplesa. Thanks for catching that.

File: docs/source/installing-brave.rst [hunk 1]
```diff
@@ -59,9 +59,9 @@ Centos/RHel
 
     sudo rpm --import https://brave-browser-rpm-release.s3.brave.com/brave-core.asc
 
-    cat << EOF | sudo tee -a /etc/yum.repos.d/Brave.repo
+    cat << EOF | sudo tee /etc/yum.repos.d/brave-browser-release.repo
```
Comment (mihaiplesa) [inline docs/source/installing-brave.rst]: this should match the package name which is `brave-browser`

File: docs/source/installing-brave.rst [hunk 2]
```diff
@@ -59,9 +59,9 @@ Centos/RHel
...
     [brave]
```
Comment (mihaiplesa) [inline docs/source/installing-brave.rst]: `brave-browser`

File: docs/source/installing-brave.rst [hunk 3]
```diff
@@ -59,9 +59,9 @@ Centos/RHel
...
-    name=Brave Browser repo
+    name=Brave Browser Release Channel repository
```
Comment (mihaiplesa) [inline docs/source/installing-brave.rst]: `Brave Browser repository`

File: docs/source/installing-brave.rst [hunk 4]
```diff
@@ -119,9 +119,9 @@ Centos/RHel
 
     sudo rpm --import https://brave-browser-rpm-beta.s3.brave.com/brave-core-nightly.asc
 
-    cat << EOF | sudo tee -a /etc/yum.repos.d/Brave.repo
+    cat << EOF | sudo tee /etc/yum.repos.d/brave-browser-beta.repo
     [brave]
```
Comment (mihaiplesa) [inline docs/source/installing-brave.rst]: `brave-browser-beta`

File: docs/source/installing-brave.rst [hunk 5]
```diff
@@ -180,9 +180,9 @@ Centos/RHel
 
     sudo rpm --import  https://brave-browser-rpm-dev.s3.brave.com/brave-core-nightly.asc
 
-    cat << EOF | sudo tee -a /etc/yum.repos.d/Brave.repo
+    cat << EOF | sudo tee /etc/yum.repos.d/brave-browser-dev.repo
     [brave]
```
Comment (mihaiplesa) [inline docs/source/installing-brave.rst]: `brave-browser-dev`

**Pull request merged**